projects/plugins/boost/app/features/optimizations/cloud-css/Cloud_CSS.php

```diff
@@ -81,4 +86,39 @@ public function display_critical_css() {
 		add_filter( 'style_loader_tag', array( $display, 'asynchronize_stylesheets' ), 10, 4 );
 		add_action( 'wp_footer', array( $display, 'onload_flip_stylesheets' ) );
 	}
+
+	/**
+	 * Create a Cloud CSS requests for provider groups.
+	 *
+	 * Initialize the Cloud CSS request. Provide $post parameter to limit generating to provider groups only associated
+	 * with a specific post.
+	 *
+	 * @param \WP_Post|null $post Post of any post type to limit provider groups.
+	 */
+	public function generate_cloud_css( $post = null ) {
+		$state            = new Critical_CSS_State( 'cloud' );
+		$source_providers = new Source_Providers();
+		if ( $post ) {
+			$state->add_request_context( $post );
+		}
+		$state->create_request( $source_providers->get_providers() );
+
+		$client = new Cloud_CSS_Request( $state );
+		return $client->request_generate();
+	}
+
+	/**
+	 * Handle regeneration of Cloud CSS when a post is saved.
+	 *
+	 * @param $post_id
```

Comment:
Parameter comments are not a requirement anymore after [we added the ignore rules](https://github.com/Automattic/jetpack/blob/master/projects/plugins/boost/.phpcs.dir.xml#L34-L39). I think there is no need to add these if you think it is for phpcs.